apps/mull-ui/src/app/components/contact-row/contact-row.scss

```diff
@@ -5,49 +5,81 @@
   margin-top: 2rem;
   justify-content: space-between;
   align-items: center;
+
   .user-information {
     display: flex;
     flex-direction: row;
+    cursor: pointer;
   }
+
   .user-profile-picture {
     width: 3.5rem !important;
     height: 3.5rem !important;
     min-height: 3.5rem !important;
   }
+
   .user-details {
     display: flex;
     padding: 0.3rem;
     flex-direction: column;
     align-self: center;
+
     .username {
       margin-top: 0.5rem;
       font-weight: normal;
       font-size: 1.125rem;
       line-height: 1.125rem;
+      width: 100%;
+      max-width: 60vw;
```

Comment:
Instead of this, I found it works better to also add a flat reduction, due to the ever-present padding and right-aligned button.

so something similar to `max-width: calc(70% - 4rem)`

See my pr for example
![image](https://user-images.githubusercontent.com/23544999/114076633-dc9f7c80-9874-11eb-9027-e978b502a6ab.png)
![image](https://user-images.githubusercontent.com/23544999/114076651-e1fcc700-9874-11eb-9923-1888441f2229.png)
![image](https://user-images.githubusercontent.com/23544999/114076723-0062c280-9875-11eb-99bd-fa538d13f324.png)



See how in the last picture its much closer to the edge than in the second picture?